client-proxy/app.env

```diff
@@ -0,0 +1,6 @@
+RETENTION_HOURS=24
```

Comment:
I would be more specific with the `*.env` file name and use `openlineage-proxy.env` or  just `openlineage.env`..

Reply:
yeah makes sense.